### DISCUSSION TITLE
Training with frozen backbone (head only)

### DISCUSSION THREAD

**Discussion by Chrisguagliano** [rwightman/efficientdet-pytorch#201] [Ideas]
Hi,

Perhaps I've missed it or unsure of how to implement it, but is there a built in option to freeze the backbone and train _only_ the heads (classification and regression networks)?

Thank you!
E

**Comment by rwightman** [comment]
@esquirel123 no, currently no builtin interface for that (on the list of possible TODOs). You'll have to manually iterate over the modules and set requires_grad=False, set BN mode, etc.
